azkaban-common/src/main/java/azkaban/trigger/Trigger.java

```diff
@@ -27,6 +27,7 @@
 
 import azkaban.utils.JSONUtils;
 
+
 public class Trigger {
 
```

Comment:
Do we need to set the members to final like submitTime, submitUser ...? They are only used by getter method.